internal/storage/withdraws/withdraw.go

```diff
@@ -0,0 +1,98 @@
+package withdraws
+
+import (
+	"context"
+	"gomarket/internal/entities"
+	"gomarket/internal/errors"
+	"gomarket/internal/storage"
+	"gomarket/internal/storage/db"
+	"time"
+)
+
+type WithDrawRepository interface {
+	CreateWithdraw(ctx context.Context, accountID string, orderID string, points float64) *errors.ErrorApp
+	GetWithdraw(ctx context.Context, accountID string) ([]entities.Withdraw, *errors.ErrorApp)
+	GetWithdrawSum(ctx context.Context, accountID string) (*float64, *errors.ErrorApp)
+}
+
+var _ WithDrawRepository = &WithDrawPG{}
+
+type WithDrawPG struct {
+	pg *db.Postgres
+}
+
+func NewAccountPG(db *db.Postgres) *WithDrawPG {
```

Comment:
перепутано имя, но в целом если в пакете один конструктор, обычно просто New достаточно
```suggestion
func New(db *db.Postgres) *WithDrawPG {
```